content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md

```diff
@@ -94,7 +94,7 @@ taking. With Pulumi's
 [Crosswalk]({{< relref "/docs/guides/crosswalk/aws" >}}) APIs it's simple
 to get at this information. Inside all the main "Crosswalk for AWS"
 modules are exposed
-[Metrics]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx/cloudwatch#Metric" >}})
+[Metrics]({{< relref "/docs/guides/crosswalk/aws/cloudwatch/#subscribing-to-cloudwatch-metrics" >}})
```

Comment:
We should definitely keep the link to the `awsx` and not change it to `aws`. The reason is, `awsx` is an extension package on top of `aws`. However, we will not have the new schema-based resource docs for `awsx`. So I don't think we need to change this link from what it is currently.

The same goes for the following packages as well:

* `policy`
* `kubernetesx` (note the `x` here.)
* `eks`
* `pulumi/pulumi`

Reply:
Oh great, this addresses some other questions I had as well :).